Assets/Scripts/UI/Style/StyleComponent.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PotentialRobot.UI.Style
+{
+    [RequireComponent(typeof(RectTransform))]
+    public abstract class StyleComponent<T, T2> : MonoBehaviour, IStyleComponent where T : Component where T2 : IStyleReference
+    {
+        [SerializeField]
+        protected T _target;
+        [SerializeField, NaughtyAttributes.Expandable]
+        protected T2 _reference;
+        public RectTransform RectTransform;
+        public IStyleReference Reference => _reference;
+
+        public void OnEnable() => Subscribe(this);
+
+        public void OnDisable() => Unsubscribe(this);
+
+        public abstract void Apply(IStyle style);
+
+        public void Subscribe(IStyleComponent component)
+        => UIStyleManager.Instance.Subscribe(component);
+
+        public void Unsubscribe(IStyleComponent component)
+        => UIStyleManager.Instance.Unsubscribe(component);
```

Comment:
Indent the second line of expression bodied methods.

Reply:
Added indents to the following lines.